localization/nb_no/commands.yaml

```diff
@@ -126,6 +136,24 @@ reloadalias:
   aliases:        [reloadalias]
   description:    'Laster inn aliaser og hjelp på nytt.'
 
+# NodeManager
+node:
+  aliases: [node]
+  description:      'Håndterer noder for serveren.'
+  sub_commands:
+    add:
+      aliases:      [ny]
+      description:  'Legger til en ny node.'
+    list:
+      aliases:      [liste, noder]
+      description:  'Lister noder.'
```

Comment:
Technically there's nothing wrong with this translation but it feels *off*. Feels kinda anglified. I would suggest changing this to "Viser en liste over nodene" or something among those lines